GVRf/Framework/framework/src/main/jni/gl/gl_render_texture.h

```diff
@@ -122,7 +122,7 @@ class GLMultiviewRenderTexture: public GLRenderTexture
     virtual bool isReady() {
         return GLRenderTexture::isReady();
     }
-    virtual bool readRenderResult(uint8_t* readback_buffer){
+    virtual bool readRenderResult(uint8_t** readback_buffer){
```

Comment:
** implies the pointer passed by the caller is supposed to be changed by the callee. And it is for Vulkan but the implementation for GL cannot because of Java_org_gearvrf_GVRViewManager_readRenderResultNative. This can be confusing.